DiamondLightSource/hyperion
# Add a check topup plan

Fixes #127 
Adds a small plan to check the topup and sleep until it's done if too soon.

Link to dodal PR (if required): [#190](https://github.com/DiamondLightSource/dodal/pull/190)

### To test:
1. Check that topup plan is called in a sensible location
2. Check that tests make sense and pass
